casepro/statistics/management/commands/calculate_existing_totals.py

```diff
@@ -0,0 +1,95 @@
+from __future__ import unicode_literals
+
+from math import ceil
+
+from django.core.management.base import BaseCommand
+from django.db import transaction
+from casepro.cases.models import Case, CaseAction, Outgoing
+from casepro.statistics.models import datetime_to_date, DailySecondTotalCount
+
+
+def calculate_totals_for_cases(case_id, progress_callback=None):
+    qs = Case.objects
+    if case_id is not None:
+        qs = qs.filter(id__gte=case_id)
```

Comment:
How I expected it to work is that if it fails after print "processed Case ID 2" they would restart it with --case 3 rather than --case 2